cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -296,11 +296,9 @@ def _ssa_single_vm():
 
         @wait_for_decorator(timeout="10m", delay=5)
         def get_ip_address():
-            ip = vm.mgmt.ip
-            logger.info("Fetched IP for %s: %s", vm_name, ip)
-            return ip is not None
+            return vm.ip_address is not None
```

Comment:
@mshriver could you please keep logger.info there .?
Else LGTM 👍

Reply:
The `ip_address` method is logging itself, but I can break this back down to log it here too if what's logged in ip_address isn't useful to you.

Reply:
Changed this up a bit more to instead use find_pingable